analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnchangedLocalVariablesShouldBeConst.cs

```diff
@@ -347,3 +348,23 @@ public NormalClass NormalClassSetToNull()
 
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/4015
```

Comment:
interesting! should the issue be only about `Expression` or , in general, classes from `System.Linq.Expressions`?